public/components/marketing-stats/3.html

```diff
@@ -0,0 +1,38 @@
+<section role="region" aria-label="Statistics" class="px-4 py-16 mx-auto flex flex-col items-center justify-center">
```

Comment:
Remove `role` and `aria-label`, they are not needed.

Reply:
There's no container, something like `max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8` is the standard.

Reply:
Don't need the `flex flex-col items-center justify-center` classes.